server/controllers/industry.js

```diff
@@ -141,8 +216,191 @@ const deleteJob = async (req, res) => {
+
+        //check if user exists
+        const user = await retrieveOrCacheUsers(req, decodeAccessToken.email); 
+
+        //check if user type is industry
+        if (user.userType.Type != process.env.INDUSTRY) {
+            return res.status(400).json(generateRes(false, 400, 'BAD_REQUEST', {}));
+        }
+
+        const industryData = await retrieveOrCacheIndustry(req, user.userType.industryData);
+        if (!industryData) {
+            return res.status(404).json(generateRes(false, 404, "INDUSTRY_DATA_NOT_FOUND", {}));
+        }
+
+        const assessments = industryData.assessments;
+
+        return res.status(200).json(generateRes(true, 200, "ASSESSMENTS_FOUND", { assessments: assessments }));
+    } catch (error) {
+        return res.status(400).json(generateRes(false, 400, "BAD_REQUEST", { error }));
+    }
+};
+
+const getAssessment = async (req, res) => {
+    try {
+        const accessToken = req.header('Authorization').split(' ')[1];
+        const decodeAccessToken = jwt.verify(accessToken, process.env.SECRET_ACCESS_TOKEN);
+
+        const assessmentId = req.params.assessmentId;
+        if (!assessmentId) {
+            return res.status(400).json(generateRes(false, 400, 'BAD_REQUEST', {}));
+        }
+
+        //check if user exists
+        const user = await retrieveOrCacheUsers(req, decodeAccessToken.email); 
+
+        //check if user type is industry
+        if (user.userType.Type != process.env.INDUSTRY) {
+            return res.status(400).json(generateRes(false, 400, 'BAD_REQUEST', {}));
+        }
+
+        const industryData = await retrieveOrCacheIndustry(req, user.userType.industryData);
+        if (!industryData) {
+            return res.status(404).json(generateRes(false, 404, "INDUSTRY_DATA_NOT_FOUND", {}));
+        }
+
+        const assessment = industryData.assessments.find(a => a.id === assessmentId);
+        if (assessment) {
+            return res.status(200).json(generateRes(true, 200, "ASSESSMENT_FOUND", { assessment: JSON.stringify(assessment) }));
+        }
+
+        return res.status(400).json(generateRes(false, 400, 'BAD_REQUEST', {}));
+    } catch (error) {
+        return res.status(400).json(generateRes(false, 400, "BAD_REQUEST", { error }));
+    }
+};
+
+const createAssessment = async (req, res) => {
+    try {
+        const accessToken = req.header('Authorization').split(' ')[1];
+        const decodeAccessToken = jwt.verify(accessToken, process.env.SECRET_ACCESS_TOKEN);
+
+        //check if user exists
+        const user = await retrieveOrCacheUsers(req, decodeAccessToken.email); 
+
+        // check if user type is industry
+        if (user.userType.Type != process.env.INDUSTRY) {
+            return res.status(400).json(generateRes(false, 400, 'BAD_REQUEST', {}));
+        }
+
+        const assessmentName = req.body.name;
+        const assessmentQuestions = req.body.questions;
```

Comment:
similar to the comment below, I'd recommend creating a Joi valdiation schema in the helpers/InputValidation/requestValidation.js for the assessmentQuestions + details + whatever else you need validated. This will ensure hte database cannot be filled with garbage or malicious data and can provide better feedback, i.e. input_error details : invalid X Y or Z